libminifi/src/utils/net/Socket.cpp

```diff
@@ -14,19 +14,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 #include "utils/net/Socket.h"
-#include "Exception.h"
-#include <cstring>
-#include <system_error>
+
 #ifdef WIN32
-#ifndef WIN32_LEAN_AND_MEAN
-#define WIN32_LEAN_AND_MEAN
-#endif /* WIN32_LEAN_AND_MEAN */
```

Comment:
Why was this removed? This macro reduces the complexity of windows headers if defined before including, and we're still including a windows header below.

Reply:
Updated in 1081ccfb9608f8f9dd7933cf48b66cb684276837